src/lib/pages/backoffice/Loan/LoanDetails/LoanDetails.js

```diff
@@ -54,6 +55,7 @@ export default class LoanDetails extends Component {
                     <Grid.Column width={13}>
                       <Container className="spaced">
                         <Container className="spaced">
+                          {data.metadata?.document_pid && <LoanOverbookedWarning documentPid={data.metadata?.document_pid} />}
```

Comment:
```suggestion
                          {data.metadata?.document_pid && <LoanOverbookedWarning documentPid={data.metadata.document_pid} />}
```
you checked already if metadata exists in the first part of the statement